Casks/j/jalview.rb

```diff
@@ -1,18 +1,18 @@
 cask "jalview" do
   arch arm: "aarch64", intel: "x64"
 
-  version "2.11.3.0"
-  sha256 arm:   "c67afbdc207db642f96454449e8594bb80aef4e0996faa2e7ddacd891638955c",
-         intel: "4faa926bc6d925bf6332eae2fc59c29d4eec61cb502509735b57a4adb5f0d5fd"
+  version "2.11.3.3"
+  sha256 arm:   "63c3b141ca5b60b5d05a4cb68c17310b4b46144fee4a574f29f4bdac90d777e5",
+         intel: "a9c6bd5885cceb99e8d56a38b8038c62f6b89c9c03181cb0c181ed96dc47cdc3"
 
   url "https://www.jalview.org/downloads/installers/release/Jalview-#{version.dots_to_underscores}-macos-#{arch}-java_8.dmg"
   name "Jalview"
   desc "Multiple sequence alignment editor, visualisor, analysis and figure generator"
   homepage "https://www.jalview.org/"
 
   livecheck do
-    url "https://www.jalview.org/getdown/release/install4j/1.8/updates.xml"
-    regex(/newVersion="(\d+(?:\.\d+)*)"/i)
+    url "https://www.jalview.org/getdown/release/1.8/release/build_properties"
+    regex(/^VERSION="(\d+(?:\.\d+)*)"/i)
```

Comment:
`build_properties` does not appear to have quotes
```suggestion
    regex(/^VERSION=(\d+(?:\.\d+)*)/i)
```